droidlet/lowlevel/hello_robot/remote/stretch_ros_move_api.py

```diff
@@ -0,0 +1,192 @@
+import time
+import math
+import threading
+import rospy
+from std_srvs.srv import Trigger, TriggerRequest
+from sensor_msgs.msg import JointState
+from control_msgs.msg import FollowJointTrajectoryGoal
+from trajectory_msgs.msg import JointTrajectoryPoint
+from geometry_msgs.msg import PoseStamped, Pose2D, PoseWithCovarianceStamped
+from nav_msgs.msg import Odometry
+import hello_helpers.hello_misc as hm
+from tf.transformations import euler_from_quaternion
+import collections
+import numpy as np
+
+class MoveNode(hm.HelloNode):
```

Comment:
I don't fully understand why we need this class - maybe it would make sense to add some comments?

Reply:
done. yes it's the Move API but via ROS.
Once the robot is driven via the ROS stuff (i.e. to use hector_slam), then we cant use the move API directly anymore and are forced to control the robot via ROS as well